app/api/router.py

```diff
@@ -46,7 +46,7 @@ async def submit_tool( data: ToolRequest, _ = Depends(key_check)):
 
 @router.post("/chat", response_model=ChatResponse)
 async def chat( request: ChatRequest, _ = Depends(key_check) ):
-    from app.features.Kaichat.core import executor as kaichat_executor
+    from features.Kaichat.core import executor as kaichat_executor
```

Comment:
Fetch these utility functions from the `app` folder.

Reply:
fixed it